conans/client/generators/cmake_find_package.py

```diff
@@ -303,7 +325,13 @@ def _find_for_dep(self, pkg_name, pkg_findname, cpp_info):
                 find_dependencies_block = ''.join("        " + line if line.strip() else line
                                                   for line in f.splitlines(True))
 
+            if pkg_filename != pkg_filename.upper():
+                set_found_filename_uppercase = 'set({}_FOUND 1)'.format(pkg_filename.upper())
+            else:
+                set_found_filename_uppercase = ''
```

Comment:
This is a different issue that should be fixed in a different pull request

Reply:
@ericLemanissier asked that I add an upper case version here. I believe it would break anything though if I did this in another PR.